common/transport/amqp/src/amqp_message.ts

```diff
@@ -88,6 +89,12 @@ export class AmqpMessage {
       amqpMessage.applicationProperties['iothub-ack'] = message.ack;
     }
 
+    if (message.diagnosticPropertyData) {
+      // Codes_SRS_NODE_IOTHUB_AMQPMSG_26_001: [If message.diagnosticPropertyData is truthy, the function shall add diagnostic information as AMQP message annotation.]
```

Comment:
>add diagnostic information [](start = 112, length = 26)

this is a little vague - i'd much rather see 2 requirements, one for each property, with expected format for each property.

Reply:
added

Reply:
please also specify the name of the property (Diagnostic-Id/Correlation-Context) and the type of the property

---
In reply to: [155454102](https://github.com/Azure/azure-iot-sdk-node/pull/158#discussion_r155454102) [](ancestors = 155454102)